docs/logs/customizing-the-sdk/README.md

```diff
@@ -30,7 +30,71 @@ TODO
 
 ### SetResourceBuilder
 
-TODO
+[Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md)
```

Comment:
this has references to "sampler" which is a tracing only concept, so need to tweak this accordingly.
Also the user is not building the provider here, instead they build the logger factory, so we need to modify wording accordingly.

Reply:
I cleaned up this section:

[Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md) is the immutable representation of the entity producing the telemetry.
If no `Resource` is explicitly configured, the default is to use a resource
indicating this [Telemetry SDK](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk).
The `SetResourceBuilder` method can be used to set a single `ResourceBuilder`.
If `SetResourceBuilder` is called multiple times, only the last is kept..
It is not possible to change the resource builder *after* creating the LoggerFactory.